### ISSUE TITLE
How can I get the tokens?

### ISSUE THREAD

**Issue by bgshin** [hanxiao/bert-as-service#117]
**Prerequisites**

> Please fill in by replacing `[ ]` with `[x]`.

* [x] Are you running the latest `bert-as-service`?
* [x] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [x] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#faq)?
* [x] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

**System information**

> Some of this information can be collected via [this script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh).

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 16.04
- TensorFlow installed from (source or binary): binary
- TensorFlow version: 1.10.0
- Python version: 3.5.2
- `bert-as-service` version: 1.5.0
- GPU model and memory: 
- CPU model and memory:

---

Thanks for the nice project!
I got the vectors by bc.encode([...]) command.
I just want to get the tokens from the input sentence.
Is there any way to do it?

Thanks

**Comment by hanxiao** [comment]
do you mean token-level embedding? see: https://github.com/hanxiao/bert-as-service/blob/master/README.md#getting-elmo-like-contextual-word-embedding

or do you mean just tokens: “something” “like” “this”, “.”?

**Comment by hanxiao** [comment]
no activity, close.

maybe related to #124 and #125

Please use the latest version 1.5.2 by
```bash
pip install -U bert-serving-client bert-serving-server
```
and follows this tutorial: https://github.com/hanxiao/bert-as-service#using-your-own-tokenizer

**Comment by bgshin** [comment]
I meant just tokens: “something” “like” “this”, “.”?

**Comment by hanxiao** [comment]
To have a full control of tokenization, please refer to:
https://github.com/hanxiao/bert-as-service#using-your-own-tokenizer

make sure you are using the latest version via
```bash
pip install -U bert-serving-server bert-serving-clinet
```

**Comment by hanxiao** [comment]
fyi, this feature (retrieving tokenization from server) is implemented in #226 and is available since 1.8.0. Please do 
```bash
pip install -U bert-serving-server bert-serving-client
```
for the update. Usage can be found in the tutorial https://github.com/hanxiao/bert-as-service#using-your-own-tokenizer or documentation: https://bert-as-service.readthedocs.io/
